packages/fe-mockserver-plugin-cds/src/index.ts

```diff
@@ -40,14 +41,24 @@ export default class CDSMetadataProvider implements IMetadataProcessor {
     }
     async loadI18nMap(dirName: string): Promise<Record<string, string>> {
         const i18nMap = { ...commonI18n };
-        if (await this.fileLoader.exists(dirName + '/i18n')) {
-            await this.loadI18nMapFolder(dirName + '/i18n/', i18nMap);
+        if (await this.fileLoader.exists(path.resolve(dirName, './i18n'))) {
+            await this.loadI18nMapFolder(path.resolve(dirName, './i18n'), i18nMap);
         }
-        if (await this.fileLoader.exists(dirName + '/_i18n')) {
-            await this.loadI18nMapFolder(dirName + '/_i18n/', i18nMap);
+        if (await this.fileLoader.exists(path.resolve(dirName, './_i18n'))) {
+            await this.loadI18nMapFolder(path.resolve(dirName, './_i18n'), i18nMap);
         }
-        if (await this.fileLoader.exists(path.resolve(dirName + '../_i18n'))) {
-            await this.loadI18nMapFolder(path.resolve(dirName + '../_i18n'), i18nMap);
+        if (await this.fileLoader.exists(path.resolve(dirName, '../_i18n'))) {
+            await this.loadI18nMapFolder(path.resolve(dirName, '../_i18n'), i18nMap);
+        }
+        let i18nPaths: string[] = this.options?.i18nPath as string[];
```

Comment:
This one actually uses metadataProcessor.options.i18nPath

Reply:
Meh i will check that :)